schedule/yam/control_center_container.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: control_center_container
+description: >
+  Install system and then launch the container with the YaST2 control center.
+vars:
+  YUI_REST_API: 1
```

Comment:
could you please set this at the job group level? we have this historically but doesn't make sense.

Reply:
we can do this later.